.github/workflows/build_and_test.yml

```diff
@@ -541,6 +541,14 @@ jobs:
           --no-rapidjson \
           --safety-profile \
           --tests
+    - name: exclude OpenDDS_Util and opendds_idl
+      if: steps.cache-artifact.outputs.cache-hit != 'true'
+      shell: bash
+      run: |
+        cd OpenDDS/build/host
+        perl -ni.bak -e "print unless /dds/" host_tools.mwc
+        . setenv.sh
+        $ACE_ROOT/bin/mwc.pl -type gnuace host_tools.mwc
```

Comment:
I'm OK with this.  Another strategy to consider is to leave the makefiles as-is but just `make` the specific targets that are needed.